node_modules/oae-messagebox/tests/test-messagebox.js

```diff
@@ -163,47 +166,53 @@ describe('Messagebox', function() {
+            var newTenantAlias = TenantsTestUtil.generateTestTenantAlias();
+            var tenantHost = 'abcdefghijklmnop.qrstuvw.xyz';
+            TestsUtil.createTenantWithAdmin(newTenantAlias, tenantHost, function(err, tenant, tenantAdminRestContext) {
                 assert.ok(!err);
 
-                // Verify the link was replaced
-                MessageBoxAPI.getMessagesFromMessageBox(messageBoxId, null, null, null, function(err, messages) {
+                var url = '/path/-Z9+&@#%=~_|!:,.;/file?query=parameter#hash';
+                var messageBoxId = util.format('msg-box-test-%s', ShortId.generate());
+                MessageBoxAPI.createMessage(messageBoxId, 'u:camtest:foo', util.format('URL: http://%s%s more', tenantHost, url), {}, function(err, message) {
                     assert.ok(!err);
-                    verifyMessage(messages[0].id, 'URL: [' + url + '](' + url + ') more', null, messages);
 
-                    // Verify multiple links
-                    var fullUrl = 'http://cambridge.oae.com' + url;
-                    MessageBoxAPI.createMessage(messageBoxId, 'u:camtest:foo', 'URLs: ' + fullUrl + ' ' + fullUrl, {}, function(err, message) {
+                    // Verify the link was replaced
+                    MessageBoxAPI.getMessagesFromMessageBox(messageBoxId, null, null, null, function(err, messages) {
                         assert.ok(!err);
+                        verifyMessage(messages[0].id, util.format('URL: [%s](%s) more', url, url), null, messages);
 
-                        // Verify the link was replaced
-                        MessageBoxAPI.getMessagesFromMessageBox(messageBoxId, null, null, null, function(err, messages) {
+                        // Verify multiple links
+                        var fullUrl = util.format('http://%s%s', tenantHost, url);
+                        MessageBoxAPI.createMessage(messageBoxId, 'u:camtest:foo', util.format('URLs: %s %s', fullUrl, fullUrl), {}, function(err, message) {
                             assert.ok(!err);
-                            var markdownFullUrl = '[' + url + '](' + url + ')';
-                            verifyMessage(messages[0].id, 'URLs: ' + markdownFullUrl + ' ' + markdownFullUrl, null, messages);
 
-                            // Verify multiple markdown links
-                            var markdownFullAbsoluteUrl = '[' + url + '](https://cambridge.oae.com' + url + ')';
-                            MessageBoxAPI.createMessage(messageBoxId, 'u:camtest:foo', 'URLs: ' + markdownFullUrl  + markdownFullUrl, {}, function(err, message) {
+                            // Verify the link was replaced
+                            MessageBoxAPI.getMessagesFromMessageBox(messageBoxId, null, null, null, function(err, messages) {
                                 assert.ok(!err);
+                                var markdownFullUrl = util.format('[%s](%s)', url, url);
```

Comment:
This just verifies that `[/path/...](/path/...)` is left unchanged when processing. However, it seems that this test was supposed to test that `[https://oaedomain/path/...](https://oaedomain/path/...)` is correctly changed to `[/path/...](/path/...)`